Build failure: lxterminal

## Steps To Reproduce

Steps to reproduce the behavior:

1. build *lxterminal*

## Build log

<!-- insert build log in code block in collapsable section -->

<details>

<summary>Build Log</summary>

```bash
this derivation will be built:
  /nix/store/cqr6gd81526mi4hyf7vhab11bpdy8zhv-lxterminal-0.4.0.drv
building '/nix/store/cqr6gd81526mi4hyf7vhab11bpdy8zhv-lxterminal-0.4.0.drv'...
Running phase: unpackPhase
unpacking source archive /nix/store/5qkd8kbhf3kdwzcd8pjgbp0jdsssb59k-source
source root is source
Running phase: patchPhase
applying patch /nix/store/c5asllmam44d46lgq91yc9229v9g8jlr-respect-xml-catalog-files-var.patch
patching file acinclude.m4
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
+ aclocal
+ autoheader --force
+ AUTOMAKE=automake intltoolize -c --automake --force
+ automake --add-missing --copy --include-deps
configure.ac:13: installing './compile'
configure.ac:8: installing './install-sh'
configure.ac:8: installing './missing'
src/Makefile.am: installing './depcomp'
+ autoconf
configure.ac:3: warning: 'AM_CONFIG_HEADER': this macro is obsolete.
configure.ac:3: You should use the 'AC_CONFIG_HEADERS' macro instead.
./lib/autoconf/general.m4:2434: AC_DIAGNOSE is expanded from...
aclocal.m4:1767: AM_CONFIG_HEADER is expanded from...
configure.ac:3: the top level
configure.ac:10: warning: The macro 'AC_PROG_INTLTOOL' is obsolete.
configure.ac:10: You should run autoupdate.
aclocal.m4:1004: AC_PROG_INTLTOOL is expanded from...
configure.ac:10: the top level
configure.ac:13: warning: The macro 'AC_ISC_POSIX' is obsolete.
configure.ac:13: You should run autoupdate.
./lib/autoconf/specific.m4:732: AC_ISC_POSIX is expanded from...
configure.ac:13: the top level
configure.ac:18: warning: The macro 'AC_HEADER_STDC' is obsolete.
configure.ac:18: You should run autoupdate.
./lib/autoconf/headers.m4:663: AC_HEADER_STDC is expanded from...
configure.ac:18: the top level
configure.ac:21: warning: The macro 'AC_HELP_STRING' is obsolete.
configure.ac:21: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
./lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.ac:21: the top level
configure.ac:28: warning: The macro 'AC_HELP_STRING' is obsolete.
configure.ac:28: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
./lib/autoconf/general.m4:1534: AC_ARG_ENABLE is expanded from...
configure.ac:28: the top level
configure.ac:36: warning: The macro 'AC_HELP_STRING' is obsolete.
configure.ac:36: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
configure.ac:36: the top level
configure.ac:87: warning: The macro 'GLIB_GNU_GETTEXT' is obsolete.
configure.ac:87: You should run autoupdate.
aclocal.m4:388: GLIB_GNU_GETTEXT is expanded from...
aclocal.m4:488: AM_GLIB_GNU_GETTEXT is expanded from...
configure.ac:87: the top level
configure.ac:87: warning: The macro 'AC_TRY_LINK' is obsolete.
configure.ac:87: You should run autoupdate.
./lib/autoconf/general.m4:2918: AC_TRY_LINK is expanded from...
lib/m4sugar/m4sh.m4:690: _AS_IF_ELSE is expanded from...
lib/m4sugar/m4sh.m4:697: AS_IF is expanded from...
./lib/autoconf/general.m4:2249: AC_CACHE_VAL is expanded from...
./lib/autoconf/general.m4:2270: AC_CACHE_CHECK is expanded from...
aclocal.m4:73: GLIB_LC_MESSAGES is expanded from...
aclocal.m4:388: GLIB_GNU_GETTEXT is expanded from...
aclocal.m4:488: AM_GLIB_GNU_GETTEXT is expanded from...
configure.ac:87: the top level
configure.ac:87: warning: The macro 'AC_TRY_LINK' is obsolete.
configure.ac:87: You should run autoupdate.
./lib/autoconf/general.m4:2918: AC_TRY_LINK is expanded from...
lib/m4sugar/m4sh.m4:690: _AS_IF_ELSE is expanded from...
lib/m4sugar/m4sh.m4:697: AS_IF is expanded from...
./lib/autoconf/general.m4:2249: AC_CACHE_VAL is expanded from...
./lib/autoconf/general.m4:2270: AC_CACHE_CHECK is expanded from...
lib/m4sugar/m4sh.m4:697: AS_IF is expanded from...
./lib/autoconf/headers.m4:89: _AC_CHECK_HEADER_COMPILE is expanded from...
./lib/autoconf/headers.m4:56: AC_CHECK_HEADER is expanded from...
aclocal.m4:172: GLIB_WITH_NLS is expanded from...
aclocal.m4:388: GLIB_GNU_GETTEXT is expanded from...
aclocal.m4:488: AM_GLIB_GNU_GETTEXT is expanded from...
configure.ac:87: the top level
configure.ac:87: warning: The macro 'AC_TRY_LINK' is obsolete.
configure.ac:87: You should run autoupdate.
./lib/autoconf/general.m4:2918: AC_TRY_LINK is expanded from...
lib/m4sugar/m4sh.m4:697: AS_IF is expanded from...
./lib/autoconf/headers.m4:89: _AC_CHECK_HEADER_COMPILE is expanded from...
./lib/autoconf/headers.m4:56: AC_CHECK_HEADER is expanded from...
aclocal.m4:172: GLIB_WITH_NLS is expanded from...
aclocal.m4:388: GLIB_GNU_GETTEXT is expanded from...
aclocal.m4:488: AM_GLIB_GNU_GETTEXT is expanded from...
configure.ac:87: the top level
configure.ac:87: warning: The macro 'AC_OUTPUT_COMMANDS' is obsolete.
configure.ac:87: You should run autoupdate.
./lib/autoconf/status.m4:1025: AC_OUTPUT_COMMANDS is expanded from...
aclocal.m4:172: GLIB_WITH_NLS is expanded from...
aclocal.m4:388: GLIB_GNU_GETTEXT is expanded from...
aclocal.m4:488: AM_GLIB_GNU_GETTEXT is expanded from...
configure.ac:87: the top level
configure.ac:96: warning: The macro 'AC_HELP_STRING' is obsolete.
configure.ac:96: You should run autoupdate.
./lib/autoconf/general.m4:204: AC_HELP_STRING is expanded from...
acinclude.m4:5: JH_PATH_XML_CATALOG is expanded from...
acinclude.m4:39: JH_CHECK_XML_CATALOG is expanded from...
configure.ac:96: the top level
configure.ac:104: warning: AC_OUTPUT should be used without arguments.
configure.ac:104: You should run autoupdate.
+ rm -rf autom4te.cache
patching script interpreter paths in ./configure
./configure: interpreter directive changed from "#! /bin/sh" to "/nix/store/5mh7kaj2fyv8mk4sfq1brwxgc02884wi-bash-5.2p37/bin/sh"
configure flags: --disable-dependency-tracking --prefix=/nix/store/33mf4l56nd7mdh9kjn6igsabmrza0lzh-lxterminal-0.4.0 --enable-man --enable-gtk3
checking whether make supports nested variables... yes
checking for a BSD-compatible install... /nix/store/4s9rah4cwaxflicsk5cndnknqlk9n4p3-coreutils-9.5/bin/install -c
checking whether build environment is sane... yes
checking for a race-free mkdir -p... /nix/store/4s9rah4cwaxflicsk5cndnknqlk9n4p3-coreutils-9.5/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether NLS is requested... yes
checking for intltool-update... /nix/store/5kg3f63gkpwgdawnv9hvygcdis4arjvg-intltool-0.51.0/bin/intltool-update
checking for intltool-merge... /nix/store/5kg3f63gkpwgdawnv9hvygcdis4arjvg-intltool-0.51.0/bin/intltool-merge
checking for intltool-extract... /nix/store/5kg3f63gkpwgdawnv9hvygcdis4arjvg-intltool-0.51.0/bin/intltool-extract
checking for xgettext... /nix/store/9a46kza0vbb4vdsdmpzbdc7f31y7hs0i-gettext-0.22.5/bin/xgettext
checking for msgmerge... /nix/store/9a46kza0vbb4vdsdmpzbdc7f31y7hs0i-gettext-0.22.5/bin/msgmerge
checking for msgfmt... /nix/store/9a46kza0vbb4vdsdmpzbdc7f31y7hs0i-gettext-0.22.5/bin/msgfmt
checking for gmsgfmt... /nix/store/9a46kza0vbb4vdsdmpzbdc7f31y7hs0i-gettext-0.22.5/bin/msgfmt
checking for perl... /nix/store/1i003ijlh9i0mzp6alqby5hg3090pjdx-perl-5.40.0/bin/perl
checking for perl >= 5.8.1... 5.40.0
checking for XML::Parser... ok
checking whether to enable maintainer-specific portions of Makefiles... yes
checking whether make supports the include directive... yes (GNU style)
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether gcc accepts -g... yes
checking for gcc option to enable C11 features... none needed
checking whether gcc understands -c and -o together... yes
checking dependency style of gcc... none
checking for library containing strerror... none required
checking for gcc... (cached) gcc
checking whether the compiler supports GNU C... (cached) yes
checking whether gcc accepts -g... (cached) yes
checking for gcc option to enable C11 features... (cached) none needed
checking whether gcc understands -c and -o together... (cached) yes
checking dependency style of gcc... (cached) none
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for grep that handles long lines and -e... /nix/store/aap6cq56amx4mzbyxp2wpgsf1kqjcr1f-gnugrep-3.11/bin/grep
checking for egrep... /nix/store/aap6cq56amx4mzbyxp2wpgsf1kqjcr1f-gnugrep-3.11/bin/grep -E
checking for locale.h... yes
checking for langinfo.h... yes
checking pkg-config is at least version 0.9.0... yes
checking for x11... yes
checking for gtk+-3.0 >= 3.0.0... yes
checking for glib-2.0 >= 2.6.0... yes
checking for vte-2.91 >= 0.20.0... yes
checking for locale.h... (cached) yes
checking for LC_MESSAGES... yes
checking for CFPreferencesCopyAppValue... no
checking for CFLocaleCopyCurrent... no
checking for libintl.h... yes
checking for ngettext in libc... yes
checking for dgettext in libc... yes
checking for bind_textdomain_codeset... yes
checking for msgfmt... (cached) /nix/store/9a46kza0vbb4vdsdmpzbdc7f31y7hs0i-gettext-0.22.5/bin/msgfmt
checking for dcgettext... yes
checking if msgfmt accepts -c... yes
checking for gmsgfmt... (cached) /nix/store/9a46kza0vbb4vdsdmpzbdc7f31y7hs0i-gettext-0.22.5/bin/msgfmt
checking for xgettext... (cached) /nix/store/9a46kza0vbb4vdsdmpzbdc7f31y7hs0i-gettext-0.22.5/bin/xgettext
checking for xsltproc... /nix/store/ckqaamnjzciyl1wgmh99cjlpdjmzvaiz-libxslt-1.1.42-bin/bin/xsltproc
checking for XML catalog (/etc/xml/catalog)... not found
checking for xmlcatalog... /nix/store/5g6g9dw135s2khr7ykn3b43i12xxzcna-libxml2-2.13.5-bin/bin/xmlcatalog
checking for DocBook XML DTD V4.1.2 in XML catalog... found
checking for DocBook XSL Stylesheets >= 1.70.1 in XML catalog... found
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating src/Makefile
config.status: creating data/Makefile
config.status: creating data/lxterminal.conf
config.status: creating po/Makefile.in
config.status: creating man/Makefile
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing default-1 commands
config.status: executing po/stamp-it commands

LXTerminal................................... : Version 0.4.0

Prefix....................................... : /nix/store/33mf4l56nd7mdh9kjn6igsabmrza0lzh-lxterminal-0.4.0
Running phase: buildPhase
build flags: SHELL=/nix/store/5mh7kaj2fyv8mk4sfq1brwxgc02884wi-bash-5.2p37/bin/bash
make  all-recursive
make[1]: Entering directory '/build/source'
Making all in src
make[2]: Entering directory '/build/source/src'
  CC       lxterminal-unixsocket.o
  CC       lxterminal-setting.o
  CC       lxterminal-lxterminal.o
lxterminal.c: In function 'terminal_copy_activate_event':
lxterminal.c:526:5: warning: 'vte_terminal_copy_clipboard' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  526 |     vte_terminal_copy_clipboard(VTE_TERMINAL(term->vte));
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vte.h:38,
                 from lxterminal.c:31:
/nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vtedeprecated.h:109:6: note: declared here
  109 | void vte_terminal_copy_clipboard(VteTerminal *terminal) _VTE_CXX_NOEXCEPT _VTE_GNUC_NONNULL(1);
      |      ^~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c: In function 'terminal_name_tab_response_event':
lxterminal.c:565:13: warning: 'vte_terminal_get_window_title' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  565 |             title = vte_terminal_get_window_title(VTE_TERMINAL(term->vte));
      |             ^~~~~
/nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vtedeprecated.h:217:13: note: declared here
  217 | const char *vte_terminal_get_window_title(VteTerminal *terminal) _VTE_CXX_NOEXCEPT _VTE_GNUC_NONNULL(1);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c: In function 'terminal_switch_page_event':
lxterminal.c:804:13: warning: 'vte_terminal_get_window_title' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  804 |             titaux=vte_terminal_get_window_title(VTE_TERMINAL(term->vte));
      |             ^~~~~~
/nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vtedeprecated.h:217:13: note: declared here
  217 | const char *vte_terminal_get_window_title(VteTerminal *terminal) _VTE_CXX_NOEXCEPT _VTE_GNUC_NONNULL(1);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c: In function 'terminal_window_title_changed_event':
lxterminal.c:825:9: warning: 'vte_terminal_get_window_title' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  825 |         gtk_label_set_text(GTK_LABEL(term->label), vte_terminal_get_window_title(VTE_TERMINAL(vte)));
      |         ^~~~~~~~~~~~~~~~~~
/nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vtedeprecated.h:217:13: note: declared here
  217 | const char *vte_terminal_get_window_title(VteTerminal *terminal) _VTE_CXX_NOEXCEPT _VTE_GNUC_NONNULL(1);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:826:9: warning: 'vte_terminal_get_window_title' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  826 |         gtk_widget_set_tooltip_text(term->label, vte_terminal_get_window_title(VTE_TERMINAL(vte)));
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vtedeprecated.h:217:13: note: declared here
  217 | const char *vte_terminal_get_window_title(VteTerminal *terminal) _VTE_CXX_NOEXCEPT _VTE_GNUC_NONNULL(1);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:830:13: warning: 'vte_terminal_get_window_title' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  830 |             gtk_window_set_title(GTK_WINDOW(term->parent->window), vte_terminal_get_window_title(VTE_TERMINAL(vte)));
      |             ^~~~~~~~~~~~~~~~~~~~
/nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vtedeprecated.h:217:13: note: declared here
  217 | const char *vte_terminal_get_window_title(VteTerminal *terminal) _VTE_CXX_NOEXCEPT _VTE_GNUC_NONNULL(1);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c: In function 'terminal_show_popup_menu':
lxterminal.c:973:5: warning: 'gtk_ui_manager_new' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  973 |     GtkUIManager * manager = gtk_ui_manager_new();
      |     ^~~~~~~~~~~~
In file included from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtk.h:284,
                 from lxterminal.c:28:
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkuimanager.h:130:16: note: declared here
  130 | GtkUIManager  *gtk_ui_manager_new                 (void);
      |                ^~~~~~~~~~~~~~~~~~
lxterminal.c:974:5: warning: 'gtk_action_group_new' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  974 |     GtkActionGroup * action_group = gtk_action_group_new("VTEMenu");
      |     ^~~~~~~~~~~~~~
In file included from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtk.h:254:
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkactiongroup.h:175:17: note: declared here
  175 | GtkActionGroup *gtk_action_group_new                     (const gchar                *name);
      |                 ^~~~~~~~~~~~~~~~~~~~
lxterminal.c:975:5: warning: 'gtk_action_group_set_translation_domain' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  975 |     gtk_action_group_set_translation_domain(action_group, GETTEXT_PACKAGE);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkactiongroup.h:252:17: note: declared here
  252 | void            gtk_action_group_set_translation_domain  (GtkActionGroup             *action_group,
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:976:5: warning: 'gtk_action_group_add_actions' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  976 |     gtk_action_group_add_actions(action_group, vte_menu_items, VTE_MENUITEM_COUNT, term->parent);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkactiongroup.h:210:17: note: declared here
  210 | void            gtk_action_group_add_actions             (GtkActionGroup             *action_group,
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:977:5: warning: 'gtk_ui_manager_insert_action_group' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  977 |     gtk_ui_manager_insert_action_group(manager, action_group, 0);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkuimanager.h:138:16: note: declared here
  138 | void           gtk_ui_manager_insert_action_group (GtkUIManager          *manager,
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:979:5: warning: 'gtk_ui_manager_new_merge_id' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  979 |     guint merge_id = gtk_ui_manager_new_merge_id(manager);
      |     ^~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkuimanager.h:186:16: note: declared here
  186 | guint          gtk_ui_manager_new_merge_id        (GtkUIManager          *manager);
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:980:5: warning: 'gtk_ui_manager_add_ui' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  980 |     gtk_ui_manager_add_ui(manager, merge_id, "/", "VTEMenu", NULL, GTK_UI_MANAGER_POPUP, FALSE);
      |     ^~~~~~~~~~~~~~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkuimanager.h:171:16: note: declared here
  171 | void           gtk_ui_manager_add_ui              (GtkUIManager          *manager,
      |                ^~~~~~~~~~~~~~~~~~~~~
lxterminal.c:986:13: warning: 'gtk_ui_manager_add_ui' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  986 |             gtk_ui_manager_add_ui(manager, merge_id, "/VTEMenu",
      |             ^~~~~~~~~~~~~~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkuimanager.h:171:16: note: declared here
  171 | void           gtk_ui_manager_add_ui              (GtkUIManager          *manager,
      |                ^~~~~~~~~~~~~~~~~~~~~
lxterminal.c:989:13: warning: 'gtk_ui_manager_add_ui' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  989 |             gtk_ui_manager_add_ui(manager, merge_id, "/VTEMenu",
      |             ^~~~~~~~~~~~~~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkuimanager.h:171:16: note: declared here
  171 | void           gtk_ui_manager_add_ui              (GtkUIManager          *manager,
      |                ^~~~~~~~~~~~~~~~~~~~~
lxterminal.c:996:5: warning: 'gtk_ui_manager_get_action' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  996 |     GtkAction * action_copy_url = gtk_ui_manager_get_action(manager, "/VTEMenu/CopyURL");
      |     ^~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkuimanager.h:155:16: note: declared here
  155 | GtkAction     *gtk_ui_manager_get_action          (GtkUIManager          *manager,
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:997:5: warning: 'gtk_ui_manager_get_action' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  997 |     GtkAction * action_open_url = gtk_ui_manager_get_action(manager, "/VTEMenu/OpenURL");
      |     ^~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkuimanager.h:155:16: note: declared here
  155 | GtkAction     *gtk_ui_manager_get_action          (GtkUIManager          *manager,
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:999:9: warning: 'gtk_action_set_visible' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  999 |         gtk_action_set_visible(action_copy_url, term->matched_url != NULL);
      |         ^~~~~~~~~~~~~~~~~~~~~~
In file included from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkactivatable.h:25,
                 from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtk.h:252:
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkaction.h:116:14: note: declared here
  116 | void         gtk_action_set_visible            (GtkAction     *action,
      |              ^~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:1000:9: warning: 'gtk_action_set_visible' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1000 |         gtk_action_set_visible(action_open_url, term->matched_url != NULL);
      |         ^~~~~~~~~~~~~~~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkaction.h:116:14: note: declared here
  116 | void         gtk_action_set_visible            (GtkAction     *action,
      |              ^~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:1004:5: warning: 'gtk_ui_manager_get_widget' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1004 |     gtk_menu_popup_at_pointer(GTK_MENU(gtk_ui_manager_get_widget(manager, "/VTEMenu")), (GdkEvent *) event);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkuimanager.h:149:16: note: declared here
  149 | GtkWidget     *gtk_ui_manager_get_widget          (GtkUIManager          *manager,
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c: In function 'terminal_vte_cursor_moved_event':
lxterminal.c:1026:13: warning: 'vte_terminal_get_window_title' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1026 |             gtk_label_set_markup(GTK_LABEL(term->label),g_strconcat("<b>* ",vte_terminal_get_window_title(VTE_TERMINAL(vte)),"</b>",NULL));
      |             ^~~~~~~~~~~~~~~~~~~~
/nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vtedeprecated.h:217:13: note: declared here
  217 | const char *vte_terminal_get_window_title(VteTerminal *terminal) _VTE_CXX_NOEXCEPT _VTE_GNUC_NONNULL(1);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c: In function 'terminal_settings_apply_to_term':
lxterminal.c:1110:5: warning: 'vte_terminal_set_allow_bold' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1110 |     vte_terminal_set_allow_bold(VTE_TERMINAL(term->vte), ! setting->disallow_bold);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vtedeprecated.h:185:6: note: declared here
  185 | void vte_terminal_set_allow_bold(VteTerminal *terminal,
      |      ^~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c: In function 'terminal_new':
lxterminal.c:1196:5: warning: 'vte_terminal_set_encoding' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1196 |     vte_terminal_set_encoding(VTE_TERMINAL(term->vte), nl_langinfo(CODESET), NULL);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vtedeprecated.h:133:10: note: declared here
  133 | gboolean vte_terminal_set_encoding(VteTerminal *terminal,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:1210:5: warning: 'vte_terminal_match_set_cursor_type' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1210 |     vte_terminal_match_set_cursor_type(VTE_TERMINAL(term->vte), ret, GDK_HAND2);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vtedeprecated.h:55:6: note: declared here
   55 | void vte_terminal_match_set_cursor_type(VteTerminal *terminal,
      |      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:1212:5: warning: 'vte_terminal_match_set_cursor_type' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1212 |     vte_terminal_match_set_cursor_type(VTE_TERMINAL(term->vte), ret, GDK_HAND2);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vtedeprecated.h:55:6: note: declared here
   55 | void vte_terminal_match_set_cursor_type(VteTerminal *terminal,
      |      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:1221:19: error: passing argument 1 of 'g_regex_unref' from incompatible pointer type [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wincompatible-pointer-types-Wincompatible-pointer-types8;;]
 1221 |     g_regex_unref(dingus1);
      |                   ^~~~~~~
      |                   |
      |                   VteRegex * {aka struct _VteRegex *}
In file included from /nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0/glib.h:78,
                 from lxterminal.c:27:
/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0/glib/gregex.h:457:71: note: expected 'GRegex *' {aka 'struct _GRegex *'} but argument is of type 'VteRegex *' {aka 'struct _VteRegex *'}
  457 | void              g_regex_unref                 (GRegex              *regex);
      |                                                  ~~~~~~~~~~~~~~~~~~~~~^~~~~
lxterminal.c:1222:19: error: passing argument 1 of 'g_regex_unref' from incompatible pointer type [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wincompatible-pointer-types-Wincompatible-pointer-types8;;]
 1222 |     g_regex_unref(dingus2);
      |                   ^~~~~~~
      |                   |
      |                   VteRegex * {aka struct _VteRegex *}
/nix/store/y8fxzs8srzd6d74a85zbpssjr8wkj9q4-glib-2.82.1-dev/include/glib-2.0/glib/gregex.h:457:71: note: expected 'GRegex *' {aka 'struct _GRegex *'} but argument is of type 'VteRegex *' {aka 'struct _VteRegex *'}
  457 | void              g_regex_unref                 (GRegex              *regex);
      |                                                  ~~~~~~~~~~~~~~~~~~~~~^~~~~
lxterminal.c:1237:5: warning: 'gtk_button_set_focus_on_click' is deprecated: Use 'gtk_widget_set_focus_on_click' instead [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1237 |     gtk_button_set_focus_on_click(GTK_BUTTON(term->close_button), FALSE);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtk.h:54:
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtkbutton.h:138:23: note: declared here
  138 | void                  gtk_button_set_focus_on_click (GtkButton      *button,
      |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:1277:9: warning: 'vte_terminal_get_window_title' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1277 |         term->label = gtk_label_new(vte_terminal_get_window_title(VTE_TERMINAL(term->vte)));
      |         ^~~~
/nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vtedeprecated.h:217:13: note: declared here
  217 | const char *vte_terminal_get_window_title(VteTerminal *terminal) _VTE_CXX_NOEXCEPT _VTE_GNUC_NONNULL(1);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:1280:5: warning: 'vte_terminal_get_window_title' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1280 |     term->label = gtk_label_new((label != NULL) ? label : vte_terminal_get_window_title(VTE_TERMINAL(term->vte)));
      |     ^~~~
/nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vtedeprecated.h:217:13: note: declared here
  217 | const char *vte_terminal_get_window_title(VteTerminal *terminal) _VTE_CXX_NOEXCEPT _VTE_GNUC_NONNULL(1);
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:1312:5: warning: 'vte_terminal_spawn_sync' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1312 |     vte_terminal_spawn_sync(
      |     ^~~~~~~~~~~~~~~~~~~~~~~
/nix/store/3x8fjngxsnzg5svrmb069f8ckds5ap1b-vte-0.78.2-dev/include/vte-2.91/vte/vtedeprecated.h:91:10: note: declared here
   91 | gboolean vte_terminal_spawn_sync(VteTerminal *terminal,
      |          ^~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c: In function 'terminal_menubar_initialize':
lxterminal.c:1372:5: warning: 'gtk_ui_manager_new' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1372 |     GtkUIManager * manager = gtk_ui_manager_new();
      |     ^~~~~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkuimanager.h:130:16: note: declared here
  130 | GtkUIManager  *gtk_ui_manager_new                 (void);
      |                ^~~~~~~~~~~~~~~~~~
lxterminal.c:1373:5: warning: 'gtk_action_group_new' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1373 |     terminal->action_group = gtk_action_group_new("MenuBar");
      |     ^~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkactiongroup.h:175:17: note: declared here
  175 | GtkActionGroup *gtk_action_group_new                     (const gchar                *name);
      |                 ^~~~~~~~~~~~~~~~~~~~
lxterminal.c:1374:5: warning: 'gtk_action_group_set_translation_domain' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1374 |     gtk_action_group_set_translation_domain(terminal->action_group, GETTEXT_PACKAGE);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkactiongroup.h:252:17: note: declared here
  252 | void            gtk_action_group_set_translation_domain  (GtkActionGroup             *action_group,
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:1377:5: warning: 'gtk_action_group_add_actions' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1377 |     gtk_action_group_add_actions(terminal->action_group, menu_items, MENUBAR_MENUITEM_COUNT, terminal);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkactiongroup.h:210:17: note: declared here
  210 | void            gtk_action_group_add_actions             (GtkActionGroup             *action_group,
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:1378:5: warning: 'gtk_ui_manager_insert_action_group' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1378 |     gtk_ui_manager_insert_action_group(manager, terminal->action_group, 0);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkuimanager.h:138:16: note: declared here
  138 | void           gtk_ui_manager_insert_action_group (GtkUIManager          *manager,
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:1380:5: warning: 'gtk_ui_manager_add_ui_from_file' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1380 |     gtk_ui_manager_add_ui_from_file (manager, PACKAGE_DATA_DIR "/lxterminal/menu.ui", NULL);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkuimanager.h:163:16: note: declared here
  163 | guint          gtk_ui_manager_add_ui_from_file    (GtkUIManager          *manager,
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:1382:5: warning: 'gtk_ui_manager_get_widget' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1382 |     terminal->menu = gtk_ui_manager_get_widget(manager, "/MenuBar");
      |     ^~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkuimanager.h:149:16: note: declared here
  149 | GtkWidget     *gtk_ui_manager_get_widget          (GtkUIManager          *manager,
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c:1385:5: warning: 'gtk_ui_manager_get_accel_group' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1385 |     terminal->accel_group = gtk_ui_manager_get_accel_group(manager);
      |     ^~~~~~~~
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/deprecated/gtkuimanager.h:147:16: note: declared here
  147 | GtkAccelGroup *gtk_ui_manager_get_accel_group     (GtkUIManager          *manager);
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lxterminal.c: In function 'lxterminal_initialize':
lxterminal.c:1726:13: warning: 'gdk_screen_get_width' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1726 |             x = gdk_screen_get_width(screen) - window_width +
      |             ^
In file included from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gdk/gdkapplaunchcontext.h:31,
                 from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gdk/gdk.h:32,
                 from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtk.h:30:
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gdk/gdkscreen.h:58:14: note: declared here
   58 | gint         gdk_screen_get_width             (GdkScreen   *screen);
      |              ^~~~~~~~~~~~~~~~~~~~
lxterminal.c:1730:17: warning: 'gdk_screen_get_height' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1730 |                 y = gdk_screen_get_height(screen) - window_height +
      |                 ^
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gdk/gdkscreen.h:60:14: note: declared here
   60 | gint         gdk_screen_get_height            (GdkScreen   *screen);
      |              ^~~~~~~~~~~~~~~~~~~~~
lxterminal.c:1743:17: warning: 'gdk_screen_get_height' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1743 |                 y = gdk_screen_get_height(gtk_window_get_screen(window)) -
      |                 ^
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gdk/gdkscreen.h:60:14: note: declared here
   60 | gint         gdk_screen_get_height            (GdkScreen   *screen);
      |              ^~~~~~~~~~~~~~~~~~~~~
lxterminal.c: In function 'terminal_settings_apply':
lxterminal.c:1778:5: warning: 'gtk_widget_is_composited' is deprecated: Use 'gdk_screen_is_composited' instead [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1778 |     terminal->rgba = gtk_widget_is_composited(terminal->window);
      |     ^~~~~~~~
In file included from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtkapplication.h:27,
                 from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtkwindow.h:33,
                 from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtkdialog.h:32,
                 from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtkaboutdialog.h:30,
                 from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtk.h:31:
/nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtkwidget.h:1247:10: note: declared here
 1247 | gboolean gtk_widget_is_composited (GtkWidget *widget);
      |          ^~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [Makefile:476: lxterminal-lxterminal.o] Error 1
make[2]: Leaving directory '/build/source/src'
make[1]: *** [Makefile:403: all-recursive] Error 1
make[1]: Leaving directory '/build/source'
make: *** [Makefile:344: all] Error 2
error: builder for '/nix/store/cqr6gd81526mi4hyf7vhab11bpdy8zhv-lxterminal-0.4.0.drv' failed with exit code 2;
       last 25 log lines:
       >    60 | gint         gdk_screen_get_height            (GdkScreen   *screen);
       >       |              ^~~~~~~~~~~~~~~~~~~~~
       > lxterminal.c:1743:17: warning: 'gdk_screen_get_height' is deprecated [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
       >  1743 |                 y = gdk_screen_get_height(gtk_window_get_screen(window)) -
       >       |                 ^
       > /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gdk/gdkscreen.h:60:14: note: declared here
       >    60 | gint         gdk_screen_get_height            (GdkScreen   *screen);
       >       |              ^~~~~~~~~~~~~~~~~~~~~
       > lxterminal.c: In function 'terminal_settings_apply':
       > lxterminal.c:1778:5: warning: 'gtk_widget_is_composited' is deprecated: Use 'gdk_screen_is_composited' instead [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
       >  1778 |     terminal->rgba = gtk_widget_is_composited(terminal->window);
       >       |     ^~~~~~~~
       > In file included from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtkapplication.h:27,
       >                  from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtkwindow.h:33,
       >                  from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtkdialog.h:32,
       >                  from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtkaboutdialog.h:30,
       >                  from /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtk.h:31:
       > /nix/store/q71pkk2b2mg8n326cs0zdlwdil2b27fl-gtk+3-3.24.43-dev/include/gtk-3.0/gtk/gtkwidget.h:1247:10: note: declared here
       >  1247 | gboolean gtk_widget_is_composited (GtkWidget *widget);
       >       |          ^~~~~~~~~~~~~~~~~~~~~~~~
       > make[2]: *** [Makefile:476: lxterminal-lxterminal.o] Error 1
       > make[2]: Leaving directory '/build/source/src'
       > make[1]: *** [Makefile:403: all-recursive] Error 1
       > make[1]: Leaving directory '/build/source'
       > make: *** [Makefile:344: all] Error 2
```

</details>

## Additional context

<!-- Add any other context about the problem here. -->

## Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.7-xanmod1, NixOS, 25.05 (Warbler), 25.05beta733374.8f3e1f807051`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.11`
 - channels(root): `"home-manager, nixos, nur"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->

## Notify maintainers

@pbsds

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/lx/lxterminal/fix-gcc14-pr122.patch
+++ b/pkgs/by-name/lx/lxterminal/fix-gcc14-pr122.patch
@@ -1,0 +1,24 @@
+diff --git a/src/lxterminal.c b/src/lxterminal.c
+index 015f5e4..32e3b68 100644
+--- a/src/lxterminal.c
++++ b/src/lxterminal.c
+@@ -1210,6 +1210,8 @@ static Term * terminal_new(LXTerminal * terminal, const gchar * label, const gch
+     vte_terminal_match_set_cursor_type(VTE_TERMINAL(term->vte), ret, GDK_HAND2);
+     ret = vte_terminal_match_add_regex(VTE_TERMINAL(term->vte), dingus2, 0);
+     vte_terminal_match_set_cursor_type(VTE_TERMINAL(term->vte), ret, GDK_HAND2);
++    vte_regex_unref(dingus1);
++    vte_regex_unref(dingus2);
+ #else
+     GRegex * dingus1 = g_regex_new(DINGUS1, G_REGEX_OPTIMIZE, 0, NULL);
+     GRegex * dingus2 = g_regex_new(DINGUS2, G_REGEX_OPTIMIZE, 0, NULL);
+@@ -1217,9 +1219,9 @@ static Term * terminal_new(LXTerminal * terminal, const gchar * label, const gch
+     vte_terminal_match_set_cursor_type(VTE_TERMINAL(term->vte), ret, GDK_HAND2);
+     ret = vte_terminal_match_add_gregex(VTE_TERMINAL(term->vte), dingus2, 0);
+     vte_terminal_match_set_cursor_type(VTE_TERMINAL(term->vte), ret, GDK_HAND2);
+-#endif
+     g_regex_unref(dingus1);
+     g_regex_unref(dingus2);
++#endif
+ 
+     /* Create a horizontal box inside an event box as the toplevel for the tab label. */
+     term->tab = gtk_event_box_new();

--- a/pkgs/by-name/lx/lxterminal/package.nix
+++ b/pkgs/by-name/lx/lxterminal/package.nix
@@ -54,6 +54,7 @@ stdenv.mkDerivation rec {
   ];
 
   patches = [
+    ./fix-gcc14-pr122.patch # manual port of https://github.com/lxde/lxterminal/pull/122
     ./respect-xml-catalog-files-var.patch
   ];
 

